pycbc/io/hdf.py

```diff
@@ -448,6 +449,22 @@ def __init__(self, trig_file, bank_file, veto_file,
             logging.info('%i triggers remain after cut on %s',
                          sum(self.mask), filter_func)
 
+    def __getitem__(self, key):
+        # Is key in the TRIGGER_MERGE file?
+        try:
+            vals = self.get_column(key)
+            return vals
+        except KeyError:
+            pass
+
+        # Is key in the bank file?
+        try:
+            self.checkbank(key)
+            return self.bank[key][:][self.template_id]
+        except RuntimeError as exc:
```

Comment:
Will both preceding indexing statements always raise `RuntimeError`s? No `KeyError`s?

Reply:
I added both exception types in here